src/components/vanilla/charts/MapChart/index.tsx

```diff
@@ -85,50 +85,52 @@ export default (props: Props) => {
             ref={tooltip}
             className="absolute text-black bg-slate-200/80 rounded-sm whitespace-nowrap pointer-events-none empty:opacity-0 opacity-100 px-2 py-1 text-xs"
           />
-          <ComposableMap
-            projection="geoMercator"
-            projectionConfig={{
-              rotate: [-10, 0, 0],
-              scale: 100
-            }}
-          >
-            <Geographies geography={geography}>
-              {({ geographies }) =>
-                geographies.map((geo) => {
-                  const value = data[geo.id] || data[geo?.properties?.name || ''];
-
-                  return (
-                    <Geography
-                      onMouseEnter={() => {
-                        tooltip.current!.innerHTML = `${geo.properties.name}: ${formatValue(
-                          `${value || 0}`,
-                          { type: 'number', meta: props.metric?.meta }
-                        )}`;
-                      }}
-                      onMouseLeave={() => {
-                        tooltip.current!.innerHTML = '';
-                      }}
-                      key={geo.rsmKey}
-                      geography={geo}
-                      style={{
-                        default: {
-                          outline: 'none'
-                        },
-                        hover: {
-                          fill: hoverColor,
-                          outline: 'none'
-                        },
-                        pressed: {
-                          outline: 'none'
-                        }
-                      }}
-                      fill={`${value ? colorScale(value) : defaultColor}`}
-                    />
-                  );
-                })
-              }
-            </Geographies>
-          </ComposableMap>
+          {ComposableMap && (
```

Comment:
@mad-raccoon This is no longer needed, was fixed in https://github.com/embeddable-hq/vanilla-components/commit/b5d5016f429bba4315de9ffd0bd123cdea949b4c